### ISSUE TITLE
Is 'onclick' specific to Checkbox?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#640]
Noted while converting Checkbox to use EnabledNode mixin for https://github.com/phetsims/sun/issues/585...

The enabledProperty listener for Checkbox is:

```js
    const enabledListener = enabled => {
      if ( enabled ) {
        this.setAccessibleAttribute( 'onclick', '' );
        this.setAccessibleAttribute( 'aria-disabled', false );
      }
      else {
        this.interruptSubtreeInput(); // interrupt interaction

        // By returning false, we prevent the a11y checkbox from toggling when the enabledProperty is false. This way
        // we can keep the checkbox in tab order and don't need to add the `disabled` attribute. See https://github.com/phetsims/sun/issues/519
        // This solution was found at https://stackoverflow.com/a/12267350/3408502
        this.setAccessibleAttribute( 'onclick', 'return false' );
        this.setAccessibleAttribute( 'aria-disabled', true );
      }

      this.pickable = enabled;
      this.opacity = enabled ? 1 : options.disabledOpacity;
    };
```

Everything except the calls to `this.setAccessibleAttribute( 'onclick'` are handled by EnabledNode or its super-mixins.   Should 'onclick' also be in EnabledNode, or is it specific to Checkbox? Searching for "this.setAccessibleAttribute( 'onclick" in the PhET code base, it is unique to Checkbox.

**Comment by pixelzoom** [comment]
Slack with @zepumph and @jessegreenberg:

Michael Kauzmann  2:47 PM
Yeah, I remember looking at that while working on EnabledNode.
I am pretty sure I tracked it back to a checkbox specific issue.

Chris Malley  2:47 PM
So leave it in an additional listener added by Checkbox ?

Michael Kauzmann  2:48 PM
I think that is what we will do, but would you mind creating a separate issue about adding that line to EnabledNode, and assigning it to JG and I?

Jesse Greenberg  2:49 PM
+1, I think I would have to review the issue to make sure it is checkbox specific

**Comment by zepumph** [comment]
I'd like to understand this better. Marking for a11y meeting for discussion.

**Comment by jessegreenberg** [comment]
@zepumph and I worked on this today - The original issue for this was https://github.com/phetsims/sun/issues/519. We did some testing and found that this was also a problem for radio buttons. Basically anywhere where interaction with PDOM would cause some native HTML attribute to change this is required. We moved it out of checkbox and into ParallelDOM's new `pdomEnabledListener` so all PDOM elements will use this. We verified that adding it generally caused no new problems by testing various things in the sun demo. Closing.
